.github/labels.yml

```diff
@@ -29,28 +29,28 @@
 - name: enhancement
   description: adds a new feature or improves on an existing one
   color: ad12cc
-- name: 'bump: major'
+- name: 'change: major'
   description: A major breaking change
   color: fcf3b8
-- name: 'bump: feature'
+- name: 'change: feature'
   description: Adds a new feature or improves on an existing one
   color: fcf3b8
-- name: 'bump: fix'
+- name: 'change: fix'
   description: Fixes a bug
   color: fcf3b8
-- name: 'bump: documentation'
+- name: 'change: documentation'
   description: Changes only affect the documentation
   color: fcf3b8
-- name: 'bump: refactor'
+- name: 'change: refactor'
   description: Changes that introduce no new features or fixes
   color: fcf3b8
-- name: 'bump: performance'
+- name: 'change: performance'
   description: Improve performance of an existing feature
   color: fcf3b8
-- name: 'bump: dependencies'
+- name: 'change: dependencies'
   description: Updates to dependancies only
   color: fcf3b8
-- name: 'change: test'
+- name: 'change: tests'
```

Comment:
The `package.json` version is not pluralised?
https://github.com/xeroxinteractive/config/pull/460/files/e141540a9f3a123fa3af801490ce7bff248b76a8..17daddaaf9d4677d2a67c1761b9a277bce1d1ff8#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R116

Reply:
Changed.